PostHog/Replay/PostHogSessionReplayConfig.swift

```diff
@@ -18,6 +18,17 @@
         /// Default: true
         @objc public var maskAllImages: Bool = true
 
+        /// Enable masking of all sandboxed system views
+        /// These may include UIImagePickerController, PHPickerViewController and CNContactPickerViewController
+        /// Experimental support
+        /// Default: true
+        @objc public var maskSandboxedViews: Bool = true
+
+        /// Enable masking of images that likely originated from user's photo library
+        /// Experimental support (UIKit only)
+        /// Default: true
+        @objc public var maskPhotoLibraryImages: Bool = true
```

Comment:
how would we mask this for swiftui? if that by enacling `maskAllSandboxedViews `? if so, I'd add it to the description.